src/compute/pre_compute_app.rs

```diff
@@ -355,6 +358,24 @@ mod tests {
         }
     }
 
+    fn start_container() -> (Container<GenericImage>, String, String) {
+        let container = GenericImage::new("kennethreitz/httpbin", "latest")
+            .with_exposed_port(80.tcp())
+            .with_wait_for(WaitFor::message_on_stderr("Listening at"))
+            .with_network("bridge")
```

Comment:
Same as this morning, this line remains optional as it is the default behavior for Docker when nothing else is specified.
We prefer to not define it and use it when there is a requirement on docker network.